Formula/molden.rb

```diff
@@ -0,0 +1,20 @@
+class Molden < Formula
+  desc "Pre- and post-processing of molecular and electronic structure"
+  homepage "http://www.cmbi.ru.nl/molden/"
+  url "ftp://ftp.cmbi.ru.nl/pub/molgraph/molden/molden5.7.tar.gz"
+  sha256 "5e7b3a8bf9251626d362b1d1dd8e4362054e6bce505c43e10254265857bb6e7d"
+  revision 1
+
+  depends_on :fortran
+  depends_on :x11
+
+  def install
+    system "make"
+    bin.install "molden", "gmolden"
+  end
+
+  test do
+    # molden is an interactive program, there is not much we can test here
+    assert_match "Molden#{version}", shell_output("#{bin}/gmolden -h")
```

Comment:
Needs a real test

Reply:
There is no way for molden not to open the GUI and expect user interaction (apart from the admittedly useless `-h`). Another option is to run `otool` on the binary and check that it is linked appropriately?